include/fmt/core.h

```diff
@@ -1394,9 +1401,11 @@ inline internal::named_arg<T, FMT_CHAR(S)> arg(const S& name, const T& arg) {
   return {name, arg};
 }
 
+/// \cond
```

Comment:
This is not needed. The ambiguity is caused by signature mismatch, not presence of extra overload.

Reply:
Ah okay. I'll change it in doc/api.rst